imageroot/actions/restore-module/50traefik

```diff
@@ -13,10 +13,23 @@ import os
 request = json.load(sys.stdin)
 renv = request['environment']
 
-configure_retval = agent.tasks.run(agent_id=os.environ['AGENT_ID'], action='configure-module', data={
-    "lets_encrypt": renv["TRAEFIK_LETS_ENCRYPT"] == "True",
+# Boolean conversion helper
+def to_bool(val): return val == "True"
+
+# Build configuration data from restored environment
+config_data = {
+    "lets_encrypt": to_bool(renv["TRAEFIK_LETS_ENCRYPT"]),
     "host": renv["TRAEFIK_HOST"],
-    "http2https": renv["TRAEFIK_HTTP2HTTPS"] == "True",
+    "http2https": to_bool(renv["TRAEFIK_HTTP2HTTPS"]),
     "ldap_domain": renv["LDAP_DOMAIN"],
-})
+    "cda_status": to_bool(renv.get("CDA_STATUS", "False")),
+    "saml_status": to_bool(renv.get("SAML_STATUS", "False")),
```

Comment:
== True convert simply when it find the string `True` to a bool value `true`

you should keep it simple